test/projects/file-generate/config/config.go

```diff
@@ -15,6 +15,7 @@
 package config
 
 const (
-	OperatorName      string = "file-generate"
-	OperatorNamespace string = "file-generate"
+	OperatorName       string = "file-generate"
+	OperatorNamespace  string = "file-generate"
+	EnableOLMSkipRange        = "true"
```

Comment:
`operator-sdk` doesn't generate this as an option, rather we embed it after the fact on a per-project basis and then read this file to generate the CSV.

/close